doorstop/cli/main.py

```diff
@@ -194,6 +194,16 @@ def main(args=None):  # pylint: disable=R0915
 
     # Parse arguments
     args = parser.parse_args(args=args)
+    try:
```

Comment:
The code works, but looking again it should be inside `run_add` instead of the common area